README.md

```diff
@@ -54,7 +56,14 @@ The following example is tested on a minikube, so that you can play with this op
 ```shell
 # apply all the necessary CRDs
 % kubectl apply -f ./deploy/crds
-% kubectl apply -f ./deploy/dependent-crdse
+% kubectl apply -f ./deploy/dependent-crds
+
+
+**Note, if you are seeing the following error, it's fine, since we kept the CR generated by operator-sdk in the deploy folder to keep consistency.**
```

Comment:
**Note:** If you see the following error, do not worry.  We kept the CR generated by operator-sdk in the Deploy folder for consistency.